integration/yarn/src/main/java/alluxio/yarn/ApplicationMaster.java

```diff
@@ -397,6 +410,37 @@ private void launchWorkerContainer(Container container) {
     }
   }
 
+  /**
+   * Checks if an Alluxio master node is already running
+   * or not on the master address given.
+   *
+   * @return true if master exists, false otherwise
+   */
+  private boolean masterExists() {
+
+    // TODO(dan): derive port from Constant
```

Comment:
Can we use `Configuration.get(PropertyKey.MASTER_WEB_PORT)`? This will default to 19999 if no configuration is set.